src/redux/reducer.type.ts

```diff
@@ -15,6 +15,19 @@ export enum UpdateTypes {
     STUDY = 'study',
 }
 
+export enum StudyIndexationStatus {
```

Comment:
I'm Ok, moving those enum here is better, but is it related to the scope of this PR ? Maybe we should have change this in another PR

Reply:
I moved it here because it was previously not an enum, it was a javascript Object
I needed it as an enum to use it as a typescript type, and you can't define enums in JS/JSX files

Reply:
damned OK